packages/buckets/package.json

```diff
@@ -32,6 +32,7 @@
     "@textile/buckets-grpc": "^1.0.8",
     "@textile/context": "^0.4.7",
     "@textile/security": "^0.1.4",
+    "@textile/threads-client": "^0.6.13",
```

Comment:
i think this used to be an issue because it would pull along a bunch of crypto stuff. i think it's fine now

Reply:
Also, since you're only using the types here, it _shouldn't_ bring in anything to bundles etc.